.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -58,8 +58,8 @@ Zebra's Rust API doesn't have any support or stability guarantees, so we keep al
 </details>
 
 - [ ] Update crate versions and do a release dry-run:
-    - [ ] `cargo release version --verbose --workspace --exclude zebrad beta`
-    - [ ] `cargo release version --verbose --package zebrad [ major | minor | patch ]`
+    - [ ] `cargo release version --execute --verbose --workspace --exclude zebrad beta`
+    - [ ] `cargo release version --execute --verbose --package zebrad [ major | minor | patch ]`
     - [ ] `cargo release publish --verbose --workspace --dry-run`
 - [ ] Commit the version changes to your release PR branch using `git`: `cargo release commit --verbose --workspace`
```

Comment:
Does commit also need execute?

```suggestion
- [ ] Commit the version changes to your release PR branch using `git`: `cargo release commit --verbose --execute --workspace`
```

Reply:
Yeah, otherwise it could ignore an error when the branch is behind remote, or commit without tracked changes: 
https://github.com/crate-ci/cargo-release/blob/master/src/steps/mod.rs#LL165C18-L172C20
https://github.com/crate-ci/cargo-release/blob/master/src/ops/git.rs#L151